test/test_files/annotations.dart

```diff
@@ -56,6 +56,10 @@ class PublicAnnotationClass {
         child2 = const PublicAnnotationClass.withAnIntAsOne();
 }
 
+class ExtendedAnnotationClass extends PublicAnnotationClass {
+  const ExtendedAnnotationClass():super();
```

Comment:
I'll have to add a test case for that; didn't think about that. I'm thinking that it will work as-is, but of course, no way to be sure without testing it.